crates/web5/src/credentials/jwt_payload_vc.rs

```diff
@@ -22,13 +22,15 @@ pub struct JwtPayloadVerifiableCredential {
     #[serde(
         rename = "issuanceDate",
         serialize_with = "serialize_optional_system_time",
-        deserialize_with = "deserialize_optional_system_time"
+        deserialize_with = "deserialize_optional_system_time",
+        default
```

Comment:
this was that serde deserialize bug -- turns out if you bring a custom deserializer then the `Option<T>` doesn't pick it up so you have to make it explicit